lib/envars.ml

```diff
@@ -181,5 +181,9 @@ let print_config ?(prefix_var_name="") f coq_src_subdirs =
   fprintf f "%sWARN=%s\n" prefix_var_name "-warn-error +a-3";
   fprintf f "%sHASNATDYNLINK=%s\n" prefix_var_name
     (if Coq_config.has_natdynlink then "true" else "false");
-  fprintf f "%sCOQ_SRC_SUBDIRS=%s\n" prefix_var_name (String.concat " " coq_src_subdirs)
-
+  fprintf f "%sCOQ_SRC_SUBDIRS=%s\n" prefix_var_name (String.concat " " coq_src_subdirs);
+  fprintf f "%sCOQ_NATIVE_COMPILER_DEFAULT=%s\n" prefix_var_name
```

Comment:
Hi @proux01, just to be sure, this new variable does not seem to be used elsewhere in the PR, but this is maybe OK?

Reply:
Sure, it's just for users that want to know the default.
(it's used in the coq-makefile tests btw)

Reply:
OK. actually I had missed the occurrences because of the additional prefix (`COQ_NATIVE_COMPILER_DEFAULT` w.r.t. `grep -q "NATIVE_COMPILER_DEFAULT=yes`). thanks @proux01!